portfolio/src/main/webapp/elsewhere.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
```

Comment:
Super neat style!

Reply:
Thank you!! Just so you know, I played around a bit and I think I fixed the image issue by removing the slash in front of images.